---
title: "Martini 3 protein models - a practical introduction to different structure bias models and their comparison"
format: html
---

<hr>

In case of issues, please contact [thallmair\@fias.uni-frankfurt.de](mailto:%20thallmair@fias.uni-frankfurt.de), [luis.borges\@ens-lyon.fr](mailto:%20luis.borges@ens-lyon.fr) or [paulo.telles_de_souza\@ens-lyon.fr](mailto:%20paulo.telles_de_souza@ens-lyon.fr).

Directly adapted from the following book chapter: ADD REF HERE.

CITE US DOI THINGIMAJIG.

<hr>

### Table of contents

-   [`Introduction`](#introduction)
-   [`Tutorial Sections`](#Tutorials)
    -   I.I:
    -   I.II:
    -   I.III:
    -   I.IV:
-   [`References`](#references)

### Introduction {#introduction}

<hr>

<!--- A detailed biophysical characterization of proteins including their structural dynamics is crucial for a broad variety of research fields ranging from molecular biology and drug development to enzyme design and environmentally friendly chemistry. In recent years, modelling techniques became more and more routine tools, offering a complementary approach to experimental techniques for studying protein biophysics. Atomistic and coarse-grained (CG) molecular dynamics (MD) simulations are among the key techniques applied in the field [1, 2]. In this chapter, we will provide a hands-on introduction to the protein models available in the fully re-parametrized general-purpose Martini 3 CG force field [3]. We assume that the reader has hands-on experience with the program package GROMACS [4], for which excellent tutorials are available[5]. --->

<!--- Martini 3 combines top-down and bottom-up coarse-graining with a general mapping of 4-to-1 non-hydrogen atoms to one regular (R) bead [3]. Small (S, 3-to-1 mapping) and tiny (T, 2-to-1 mapping) are also available. The non-bonded interactions are parametrized based on experimental thermodynamic data, while the bonded interactions are parametrized using atomistic[6–8] or quantum mechanical reference simulations [7, 9]. We refer the reader for more details to some recent reviews [10, 11]. --->

The Martini 3 protein model comprises two layers, which are strictly separated from each other: The first layer contains the mapping and chosen chemical bead types as well as the bonded terms; the second layer is the structure bias model. While the first layer corresponds to the standard definition of any molecule in the Martini universe, the second layer is more specific for proteins: To maintain the secondary, tertiary and quaternary structure of proteins, the directionality of interactions, in particular hydrogen bonds, are crucial. Due to the spherical potentials used in Martini 3 and many other CG force fields, this directionality is lost. Therefore, a structure bias model is required to stabilize protein structures. The strict separation between the two layers in Martini 3 enables independent development on both layers and is in contrast to the Martini 2 model\[12\], where mappings, bead types, and bonded terms were specific to each of the protein models. The first layer of the current Martini 3 protein model is mostly inherited from the Martini 2 model with moderate updates concerning the new mapping guidelines, bead types, and sizes. The Martini 3 mappings and bead types of the proteinogenic amino acids are depicted in Figure 1. A key difference to the previous model is that the bead type of the backbone (BB) bead does not depend on the secondary structure anymore but is represented by a P2 bead. Exceptions are  the special cases glycine (SP1), alanine (SP2), valine (SP2), proline (SP2a), and terminal beads (Q5 for charged and P6 for neutral termini) \[13\].

Here, we will focus on the setup, fine-tuning options, and evaluation of the second layer of the Martini 3 protein model – the structure bias model – as well as on dedicated options for intrinsically disordered regions (IDRs). Overall, the Martini protein models require specific bonded terms to model the secondary structure \[10, 12\], which is in most cases combined with side-chain dihedral corrections \[14\]**.** In addition, there are three main options for structural bias models in layer 2 available, typically used to stabilize tertiary and quaternary structures, namely Elastic Network (EN) \[3, 15, 16\], GōMartini \[13\], and OLIVES \[17\] (Figure 2). Whereas the ad-hoc EN approach provides the most robust and straightforward way of stabilizing a protein structure, and has traditionally been used in Martini, the bioinformatics-based GōMartini approach is nowadays the method of choice as it has proven to be a more versatile method, striking a balance between protein stability and flexibility. Most recently, OLIVES was introduced as a physics-based variant of GōMartini, with the particular prospect of biasing quaternary structures as well.

In their simplest form, each of these three structure bias models relies on an atomistic reference, structure which can be obtained from experiments or prediction software. The EN model uses harmonic potentials between the BB beads within a cutoff distance to maintain the protein structure. In contrast, both the GōMartini and OLIVES models employ Lennard-Jones potentials in a Gō-type manner, allowing for contact dissociation. In the GōMartini model, the potentials are defined from a contact map based on native contacts, evaluated through overlap and restricted contacts of structural units \[18, 19\], combined with a distance cutoff. The OLIVES model, on the other hand, defines its bias by evaluating the hydrogen bonding interactions between BB as well as side chain (SC) beads \[17\]. For small structured systems, such as single α-helices, the waiver of a structural bias model can be an option as well.

The tutorial is structured as follows:

-   Tutorial I.I provides details on how to set up a Martini 3 protein model, taking a protein kinase as our workhorse, and using either EN, GōMartini, or OLIVES, together with secondary structure-specific bonded terms and side-chain corrections.

-   The evaluation of the performance of the model in comparison to atomistic reference simulations is discussed in Section Tutorial I.II.

-   Tutorial I.III gives an example for the addition of an IDR to the structured domain of the kinase.

-   Additional notes and limitations are discussed in Section 5, before we conclude in Section 6.

    <!--- Related Martini-based tutorial chapters are available for constructing PEGylated proteins [20], embedding membrane proteins [21], and for protein-ligand binding [7].  --->

## Tutorial Sections {#Tutorials}

-   [Tutorial I.I: Setup of Martini 3 Protein Structure Models](Tut1.qmd)

-   [Tutorial I.II: Metrics to Compare Protein Structure Models](Tut2.qmd)

-   [Tutorial I.III: Simulations of IDRs in Martini 3](Tut3.qmd)

-   Tutorial I.IV: Additional notes, limitations & conclusion.

### References {#references}

<hr>

<sub> \[1\] P. C. T. Souza, et al., Martini 3: a general purpose force field for coarse-grained molecular dynamics. Nat. Methods 18, 382–388 (2021). </sub>